justfile

```diff
@@ -80,38 +80,19 @@ clean:
     cd {{frb_flutter}}/rust && cargo clean
 
 check:
+    cd frb_rust && cargo clippy -- -D warnings
+    cd frb_rust && cargo clippy --target wasm32-unknown-unknown -- -D warnings
+    cd frb_codegen && cargo clippy -- -D warnings
     cd {{frb_pure}}/dart && dart pub get && dart analyze
     cd {{frb_pure}}/rust && cargo clippy
     cd {{frb_pure_multi}}/dart && dart pub get && dart analyze
     cd {{frb_pure_multi}}/rust && cargo clippy
     cd {{frb_flutter}} && flutter pub get && flutter analyze
     cd {{frb_flutter}}/rust && cargo clippy
-    cd frb_codegen && cargo clippy -- -D warnings
-    cd frb_rust && cargo clippy -- -D warnings
-    cd frb_rust && cargo clippy --target wasm32-unknown-unknown -- -D warnings
 
 serve *args:
     cd {{invocation_directory()}} && dart run {{justfile_directory()}}/frb_dart/bin/serve.dart {{args}}
 
-refresh_all:
-    just gen-help
-
-    just gen-bridge
-    (cd frb_rust && cargo clippy -- -D warnings)
-    (cd frb_macros && cargo clippy -- -D warnings)
-    (cd frb_example/pure_dart/rust && cargo clippy -- -D warnings)
-    (cd frb_example/pure_dart_multi/rust && cargo clippy -- -D warnings)
-    (cd frb_example/with_flutter/rust && cargo clippy -- -D warnings)
-    (cd frb_example/pure_dart/dart && dart pub get)
-    (cd frb_example/pure_dart_multi/dart && dart pub get)
-    (cd frb_example/with_flutter && flutter pub get)
-
-    just lint
-
-    sed -i "" -e 's/pub.flutter-io.cn/pub.dartlang.org/g' frb_example/pure_dart/dart/pubspec.lock
-    sed -i "" -e 's/pub.flutter-io.cn/pub.dartlang.org/g' frb_example/pure_dart_multi/dart/pubspec.lock
-    sed -i "" -e 's/pub.flutter-io.cn/pub.dartlang.org/g' frb_example/with_flutter/pubspec.lock
```

Comment:
hmm imho `refresh_all` contains some more scripts than `precommit`. So maybe we should add those scripts to `precommit` when we are merging refresh_all into that

Reply:
As I wrote it above that URL is not valid. I think it was pointing to something else before but now it's not. Trying to open that URL leads to 404 error page.
<img width="457" alt="image" src="https://user-images.githubusercontent.com/66480156/217835276-11932c63-182e-4714-b997-627351c38eba.png">

Reply:
Oh it means: replaces all the `pub.flutter-io.cn` text into `pub.dartlang.org` text

Reply:
I get the purpose of that domain, but that `sed -i` line produces an error probably because the resource was removed.

Reply:
Okay I will

Reply:
P.S. I did change the text and did an experiment, but error still remains.
<img width="474" alt="image" src="https://user-images.githubusercontent.com/66480156/217845332-b3fcb9a6-6603-4f7f-9f26-b38656a195cd.png">

Reply:
I will check that. btw that is probably b/c I use that on MacOS while you are linux/win?

Reply:
Hmm I'm using Windows, I'll try changing the directory slash from / to \\

Reply:
<img width="469" alt="image" src="https://user-images.githubusercontent.com/66480156/217848011-6bf2ff69-818b-4270-a51e-76788565d65d.png">

Reply:
I think the web URL is causing the problem

Reply:
> Anyway, just copy those lines and _comment them out_. I will check them later
Okay it's done